### PR TITLE
Sync wallaby into xena

### PR DESCRIPTION
**Pull request by jovial**
- [x] Build images
- [x] Promote images

### CODE REVIEW

**Comment by jovial** [comment]
Image builds:
- centos: https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/4469268391
- ubuntu: https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/4469274983

File: etc/kayobe/kolla/globals.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ blazar_tag: xena-20230315T122920
 caso_tag: xena-20230315T122920
 ironic_tag: xena-20230316T162305
 ironic_dnsmasq_tag: xena-20230214T150257
-neutron_tag: xena-20230307T142413
+neutron_tag: FIXME-BUILD-NEW-IMAGE
```
Comment (jovial) [inline etc/kayobe/kolla/globals.yml]:
```suggestion
neutron_tag: xena-20230320T143013
```

File: etc/kayobe/kolla/globals.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ caso_tag: xena-20230315T122918
 ironic_tag: xena-20230317T090705
 ironic_dnsmasq_tag: xena-20230215T164524
 keystone_tag: xena-20230308T120251
-neutron_tag: xena-20230307T142414
+neutron_tag: FIXME-BUILD-NEW-IMAGE
```
Comment (jovial) [inline etc/kayobe/kolla/globals.yml]:
```suggestion
neutron_tag: xena-20230320T143007
```

**Comment by jovial** [comment]
Promotion jobs:
- centos: https://github.com/stackhpc/stackhpc-release-train/actions/runs/4477428474
- ubuntu: https://github.com/stackhpc/stackhpc-release-train/actions/runs/4477433908

**Review by markgoddard — DISMISSED** [review]

**Comment by jovial** [comment]
Merged in CI fix for centos stream.

**Review by markgoddard — APPROVED** [review]

**Pull request merged**